spec/presenters/summary/html_presenter_spec.rb

```diff
@@ -4,11 +4,16 @@
   subject { described_class.new(crime_application:) }
 
   let(:crime_application) do
-    instance_double(CrimeApplication, applicant: double, case: double)
+    instance_double(CrimeApplication, applicant: double, case: kase)
   end
 
+  let(:kase) { instance_double(Case, ioj:) }
+
+  let(:ioj) { instance_double(Ioj) }
+
   describe '#sections' do
     before do
+      allow(kase).to receive(:ioj).and_return(ioj)
```

Comment:
Another tiny thing, no need for this allow.

Reply:
👍